Añadido top 3 y corregido botón "Continuar"

xaaniyaro opened the pull request:
- Se añadió una sección en la pantalla de menú principal para mostrar la clasificación por turnos
- Se modificó el modelo de datos para agregar "turno" como atributo con los posibles valores "Matutino", "Vespertino", "Nocturno"
- Se modificó la posición y el tamaño del botón continuar en todas las escenas
- Se añadió un icono para mostrar cuál es el turno en el que se encuentra el usuario

xaaniyaro committed:
Probando cambios

Se modificó el modelo de datos, PERO aquí  en GitHub no se va a modificar el database.json que tiene toda la información, espero que no cause errores.

xaaniyaro committed:
Añadido top 3

Se añadió el top 3 en la pantalla de Menú, muestra sólo el nombre del turno, también se añadió un icono que muestra el turno en el que se encuentra el usuario actual

xaaniyaro committed:
Podium y turnoImg no desaparecen

Se cambió la jerarquía de los objetos para que no se vieran encima del objeto oculto que mostraba la pantalla de carga de una misión

xaaniyaro committed:
Botón "Continuar" incorrectamente posicionado

Se modificó el prefab del botón, para que cuando se instanciara ocupara menos lugar de la pantalla y no se empalmara con los demás elementos, igualmente que quedara más simétrico

xaaniyaro committed:
Merge branch 'developmentSprint1' into ivanBranchSprint1

napo-lazo commented:
El podium se sale un poco de la pantalla y falta una comma en el archivo de users.json

xaaniyaro committed:
Correciones podium, json, iconos

Corregidos los anchors del Podium GameObject y del Turno GameObject.
Corregido el users.json, le faltaba una coma, cambiados los iconos del podium para que fueran del mismo tamaño y más vistosos
